dali/server/daserver.cpp

```diff
@@ -393,6 +393,49 @@ static StringBuffer &formatDaliRole(StringBuffer &out, unsigned __int64 role)
     return out.append(queryRoleName((DaliClientRole)role));
 }
 
+#ifdef _CONTAINERIZED
+static IPropertyTree * getContainerLDAPConfiguration(const char * authMethod, IPropertyTree *appConfig)
+{
+    if (streq(authMethod, "none"))
+        return nullptr; //no security manager
```

Comment:
I'd also like to see this make sure that it was set to "ldap", rather than assume not having "none" means ldap.  In the future we will have more options so we should verify.  Probably fail if it's something we don't understand.